apps/web/src/components/forms/password.tsx

```diff
@@ -21,9 +21,11 @@ import { FormErrorMessage } from '../form/form-error-message';
 export const ZPasswordFormSchema = z
   .object({
     currentPassword: z
-      .string()
-      .min(6, { message: 'Password should contain at least 6 characters' })
-      .max(72, { message: 'Password should not contain more than 72 characters' }),
+      .union([
+        z.string().min(6, { message: 'Password should contain at least 6 characters' }),
+        z.string().max(72, { message: 'Password should not contain more than 72 characters' }),
+      ])
+      .optional(),
```

Comment:
This still feels clunky, would it work as a nullable like you've done at the bottom of this PR?

Reply:
What Zod validations should I add there to make it work?
```
currentPassword: z
      .string()
      .min(6, { message: 'Password should contain at least 6 characters' })
      .max(72, { message: 'Password should not contain more than 72 characters' })
      .nullable(),
  ```
  This gives error